### PR TITLE
Mermaid

### PR DESCRIPTION
**Pull request by ascholerChemeketa**
Adds mermaid diagrams

Guide has one and sample article has a few.

### CODE REVIEW

File: pretext/pretext.cfg [hunk 1]
```diff
@@ -58,6 +58,7 @@ pdflatex = pdflatex
 xelatex = xelatex
 pdfsvg = pdf2svg
 asy = asy
+mermaid = mmdc
```
Comment (StevenClontz) [inline pretext/pretext.cfg:61]: CLI will need updates at e.g. https://github.com/PreTeXtBook/pretext-cli/blob/0fbcef135b56e92c9f6e89d75175848973fa6139/pretext/project/xml.py#L11

Comment (ascholerChemeketa) [reply pretext/pretext.cfg:61]: CLI PR is https://github.com/PreTeXtBook/pretext-cli/pull/750

Comment (StevenClontz) [reply pretext/pretext.cfg:61]: Somehow I missed that part of the PR, great!

**Comment by rbeezer** [comment]
Very good!  Looking forward to using this, some day.

* Added entries for the simple interface in `pretext/pretext`.
* Built images for static formats: sample article, The Guide.  And committed to the repository.

Thanks!

**Pull request closed without merging**